config/locales/idv/en.yml

```diff
@@ -103,7 +103,7 @@ en:
             button: Verify by mail
           heading: 'We couldn’t verify your identity by phone'
           option_try_again_later_html: 'Cancel and start over again after <b>%{time_left}</b>'
-          option_verify_by_mail_html: 'Verify by mail, which typically takes <b>3-7 business days</b>'
+          option_verify_by_mail_html: 'Verify by mail, which takes <b>5 to 10 days</b>'
```

Comment:
Nit: can you make this `<strong>`?

Reply:
Sure, I'll update this one. There are lots of other <b> </b> in the translations, and I wonder if we want to do a separate PR to update the rest.

Reply:
Added [LG-11043](https://cm-jira.usa.gov/browse/LG-11043) to update the rest of the `<b>` tags.